Makefile

```diff
@@ -162,6 +232,12 @@ release:	check-go-vars
 	cp $(GOBIN)/expenv.exe $(RELTMPDIR)
 	cp $(PGOROOT)/examples/pgo-bash-completion $(RELTMPDIR)
 	tar czvf $(RELFILE) -C $(RELTMPDIR) .
-default:
-	all
 
+
+#======= Misc =======
+deepsix:
+	cd $(PGOROOT)/apis/cr/v1
+	deepcopy-gen --go-header-file=$(PGOROOT)/apis/cr/v1/header.go.txt --input-dirs=.
+
+
+default: help
```

Comment:
My (strong) preference is that the `default` build target is not help, but remains `all`. There will be some upcoming work that continues to improve upon this.